content/cloud/variables.md

```diff
@@ -0,0 +1,191 @@
+title = "Configuring Spin Application Variables and Secrets"
+date = "2023-06-05T16:40:01.672734Z"
+template = "cloud_main"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/variables.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Creating Our Spin Application](#creating-our-spin-application)
+- [Adding Variables to an Application Component](#adding-variables-to-an-application-component)
+- [Using Variables in a Spin Application](#using-variables-in-a-spin-application)
+- [Deploying the Application to Fermyon Cloud](#deploying-the-application-to-fermyon-cloud)
+
+Spin supports dynamic application variables. Instead of being static, their values can be updated without modifying the application, creating a simpler experience for rotating secrets, updating API endpoints, and more. 
+
+These variables are defined in a Spin application manifest (in the `[variables]` section) and are provided by a configuration provider. When using Spin locally, the configuration provider can be Vault for secrets or host environment variables. You can refer to the [dynamic configuration documentation](../spin/dynamic-configuration.md) to learn how to configure variables locally. In Fermyon Cloud, you can set and update variables for Spin applications using the spin cloud variables CLI.
+
+This tutorial will guide you through the process of creating a simple application that validates passwords.
+
+## Prerequisites
+
+Before starting, ensure that you have Spin installed on your computer. You can follow the official Fermyon Cloud Quickstart guide to [install Spin](/cloud/quickstart#install-spin) and [log into Fermyon Cloud](/cloud/quickstart#log-in-to-the-fermyon-cloud).
+
+Since this example is written in Python, make sure you have the required tools installed for Python Spin applications. The Spin CLI facilitates the creation of new Spin applications using application [templates](/cloud/cli-reference#templates). In this tutorial, we will use the `http-py` template. Install it along with the `py2wasm` Spin plugin, which enables compiling Python apps to Wasm, using the following commands:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin templates install --git https://github.com/fermyon/spin-python-sdk --upgrade
+$ spin plugins install py2wasm --yes
```

Comment:
Don't we need a `spin plugins update` before this?

Reply:
definitely a good addition to make sure they are up to date